scipy/sparse/csgraph/meson.build

```diff
@@ -25,6 +24,15 @@ foreach pyx_file: pyx_files
   )
 endforeach
 
+py3.extension_module('_shortest_path',
+  cython_gen_cpp.process('_shortest_path.pyx'),
+  cpp_args: cython_cpp_args,
+  include_directories: inc_np,
+  dependencies: [py3_dep],
```

Comment:
The change here from `np_dep` to `py3_dep` does not look correct (`py3_dep` is always implicitly added already, and numpy is a dependency here). 

Also, `link_args` is dropped compared to the original code here.